CONTRIBUTING.md

```diff
@@ -14,14 +14,15 @@ the maintenance team upon request.
 <!-- TOC start -->
 - [Quick guide](#quick-guide)
 - [Best practice and recommendations](#best-practice-and-recommendations)
-- [Resources](#resources)
+- [Resources & Further Reading](#resources--furhter-reading)
```

Comment:
Typo `furhter` instead of `further`

Reply:
Good eyes. You should do PR reviews more often here. Feel invited. ;)